lib/icingadb/icingadb-objects.cpp

```diff
@@ -2589,6 +2593,13 @@ void IcingaDB::StateChangeHandler(const ConfigObject::Ptr& object, const CheckRe
 	}
 }
 
+void IcingaDB::ReachabilityChangeHandler(const Checkable::Ptr& checkable)
+{
+	for (const IcingaDB::Ptr& rw : ConfigType::GetObjectsByType<IcingaDB>()) {
+		rw->SendStatusUpdate(checkable);
```

Comment:
Doesn't that function only write a runtime update but not update the actual objects in Redis?

Reply:
Does that matter?